App/Services/EventHandlers/TextileNodeEventHandler.ts

```diff
@@ -19,14 +20,15 @@ export default class TextileNodeEventHandler {
     TextileNode.eventEmitter.addListener('onOnline', () => {
       this.store.dispatch(TextileNodeActions.nodeOnline())
     })
-    TextileNode.eventEmitter.addListener('onThreadUpdate', (payload) => {
-      this.store.dispatch(TextileNodeActions.getPhotoHashesRequest(payload.thread_id))
+    TextileNode.eventEmitter.addListener('onThreadUpdate', (update: Update) => {
+      const { type } = update.block
+      if (type === BlockType.CommentBlock || type === BlockType.LikeBlock || type === BlockType.PhotoBlock) {
+        this.store.dispatch(PhotoViewingActions.refreshThreadRequest(update.thread_id))
+      }
     })
-    TextileNode.eventEmitter.addListener('onThreadAdded', () => {
-      this.store.dispatch(ThreadsActions.refreshThreadsRequest())
+    TextileNode.eventEmitter.addListener('onThreadAdded', (payload) => {
     })
-    TextileNode.eventEmitter.addListener('onThreadRemoved', () => {
-      this.store.dispatch(ThreadsActions.refreshThreadsRequest())
+    TextileNode.eventEmitter.addListener('onThreadRemoved', (payload) => {
```

Comment:
There is really nothing to do in responding to thread add/remove events. The app redux already knows about those events because the app is the thing that created/removed the thread. @sanderpick I guess this is the only thing I wanted to be sure is ok.